packages/client/src/open-feature.ts

```diff
@@ -190,15 +191,25 @@ export class OpenFeatureAPI extends OpenFeatureCommonAPI<Provider, Hook> impleme
   }
 
   private async runProviderContextChangeHandler(
+    clientName: string | undefined,
```

Comment:
This method now also runs the new associated events (which is why we take the client name now).